app/src/main/java/org/onionshare/android/ui/share/ShareBottomSheet.kt

```diff
@@ -174,8 +177,12 @@ fun BottomSheet(state: ShareUiState, onSheetButtonClicked: () -> Unit) {
             }
             Divider(thickness = 2.dp)
         }
+        var buttonEnabled by remember(state) { mutableStateOf(true) }
         Button(
-            onClick = onSheetButtonClicked,
+            onClick = {
+                buttonEnabled = false
```

Comment:
Where does this get set back to `true`? Is it happening as a side effect of the bottom sheet being replaced?

Reply:
That's because `var buttonEnabled by remember(state) { mutableStateOf(true) }` which depends on `state`, so for a different state this will be true again.